13:4: not a valid test operator: (
13:4: not a valid test operator: 525.85.12
Downloading: "https://download.pytorch.org/models/googlenet-1378be20.pth" to /home/gharatappeh/.cache/torch/hub/checkpoints/googlenet-1378be20.pth
  0%|          | 0.00/49.7M [00:00<?, ?B/s]  4%|▍         | 1.98M/49.7M [00:00<00:02, 20.1MB/s] 12%|█▏        | 5.98M/49.7M [00:00<00:01, 32.8MB/s] 20%|█▉        | 9.91M/49.7M [00:00<00:01, 35.4MB/s] 29%|██▊       | 14.3M/49.7M [00:00<00:00, 39.4MB/s] 37%|███▋      | 18.3M/49.7M [00:00<00:00, 40.5MB/s] 45%|████▌     | 22.5M/49.7M [00:00<00:00, 41.4MB/s] 53%|█████▎    | 26.4M/49.7M [00:00<00:00, 39.6MB/s] 61%|██████▏   | 30.5M/49.7M [00:00<00:00, 40.4MB/s] 71%|███████   | 35.3M/49.7M [00:00<00:00, 43.6MB/s] 80%|████████  | 39.8M/49.7M [00:01<00:00, 44.5MB/s] 89%|████████▉ | 44.4M/49.7M [00:01<00:00, 45.6MB/s] 98%|█████████▊| 48.9M/49.7M [00:01<00:00, 46.1MB/s]100%|██████████| 49.7M/49.7M [00:01<00:00, 41.9MB/s]
  0%|          | 0/10 [00:00<?, ?it/s]  0%|          | 0/10 [06:43<?, ?it/s]
Traceback (most recent call last):
  File "/home/gharatappeh/nips/src/prune.py", line 767, in <module>
    main()
  File "/home/gharatappeh/nips/src/prune.py", line 757, in main
    perf_exper(logger, args, device, run_dir)
  File "/home/gharatappeh/nips/src/prune.py", line 715, in perf_exper
    all_acc, conn = perf_lth(logger, device, args, controller)
  File "/home/gharatappeh/nips/src/prune.py", line 624, in perf_lth
    pruning.corrs.append(activations.get_corrs())
  File "/home/gharatappeh/nips/src/correlation.py", line 177, in get_corrs
    child_arr.append(self.activation[act_keys[idx + 1]].\
IndexError: list index out of range
